README.md

```diff
@@ -114,61 +121,144 @@ The milestone needs testing on all platforms. Please report issues when found.
         --add Microsoft.VisualStudio.Component.VC.ATLMFC
     ```
 
-2. Open a “x64 Native Tools Command Prompt” and run:
+    **NOTE:**
+    Builds _must_ be performed in one of the Visual Studio-provided command prompts ("Developer Command Prompt", "Developer PowerShell", "x64 Native Tools Command Prompt", etc), _not_ a standard Command Prompt or PowerShell. The VS prompts will already have vcpkg installed and configured for use, so no dependency management is needed.
+
+2. **Clone the Descent3 source code.**
 
+    Open a Developer Command Prompt (or Developer PowerShell, x64 Native Tools Command Prompt, etc if you want a specific environment) and run:
```

Comment:
Why did we switch from telling people to use an x64 Native Tools Command Prompt to telling people to use any of the “-for VS 2022” command prompts?

Reply:
Hubris! I wanted to make sure it was clear that PowerShell would work as well, and overreached. I've switched the README back to saying
> Open a "x64 Native Tools Command Prompt" or "x64 Native Tools PowerShell" and run:

Reply:
I don’t have an x64 Native Tool PowerShell shortcut in my Start menu:

![My Start menu has an entry named “Visual Studio 2022”. Beneath that entry, there’s a folder that’s also named “Visual Studio 2022”. In that folder there are several entries: “Debuggable Package Manager”, “Developer Command Prompt for VS 2022”, “Developer PowerShell for VS 2022”, “x64 Native Tools Command Prompt for VS 2022”, “x64_x86 Cross Tools Command Prompt for VS 2022”, “x86 Native Tools Command Prompt for VS 2022” and “x86_64 Cross Tools Command Prompt for VS 2022”. Beneath all of the entries in the “Visual Studio 2022” folder is an entry named “Visual Studio Installer”.](https://github.com/user-attachments/assets/40e4b041-a56c-4948-b46f-1cd321442aba)

How do I open an x64 Native Tools PowerShell? Do I need to do something in order to get a Start menu shortcut for it?